java/java.lsp.server/nbcode/integration/src/org/netbeans/modules/nbcode/integration/standard-cookies.contextValues

```diff
@@ -0,0 +1,25 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# Use fully qualified names, one class per line. Syntax is:
+#   qualified.name=value    : value will be used instead of qualified name.
+#   [prefix.]qualified.name : the prefix will be stripped
+#   [prefix=subst]          : the prefix in all classes will be replaced by subst, possibly empty
+#   * as the first character: watch for changes in this interface, fire changes.
+# Names will be converted to "class:<name>" tokens.
+
+org.netbeans.api.actions.Openable
```

Comment:
None of the descriptions at line 18-21 seem to match `org.netbeans.api.actions.Openable` - I have no idea what it does.

Reply:
nodes that can be 'opened' somehow will manifest this using `class:Openable` at LSP client side.

Reply:
OK, let's leave it as it is for now.